packages/big-design-theme/src/helpers/helpers.ts

```diff
@@ -7,6 +8,7 @@ export interface Helpers {
   createRGBA(color: string, alpha: number): string;
   remCalc(value: string | number): string;
   emCalc(value: string | number): string;
+  listReset: FlattenSimpleInterpolation;
```

Comment:
Is this the `type` returned when it simply returns the styles via `css`?

Reply:
Yeah, `css` returns a `ReadonlyArray<...>` of values. In this case it's a `ReadonlyArray<string>`. styled-components will be able to take care of that output.